packages/backend/src/peripherals/database/types.ts

```diff
@@ -10,12 +10,14 @@ declare module 'knex/types/tables' {
 
   interface PriceRow {
     coingecko_id: string
+    asset_id?: string
```

Comment:
Why is this optional?

Reply:
I just wanted to stay without TS errors everywhere